docs/sources/alerting/rules.md

```diff
@@ -48,6 +48,7 @@ panels as well in a future release.
 ### Name and Evaluation interval
 
 Here you can specify the name of the alert rule and how often the scheduler should evaluate the alert rule.
+Note: A minimum interval can be specified in the `alerting.min_interval_seconds` config field, to set a minimum time between evaluations.
```

Comment:
```suggestion
**Note:** You can set a minimum interval in the `alerting.min_interval_seconds` config field, to set a minimum time between evaluations.
```

Reply:
Try to address instructions to "you" rather than writing them in the passive voice. They tend to be clearer and require fewer words.

Reply:
Given my other comment would suggest linking to [configuration documentation](https://grafana.com/docs/grafana/latest/installation/configuration/#alerting) of this property.

Reply:
Good point, @marefr . That would be a good improvement.